### PR TITLE
Formatted output of date and time variables

### PR DESCRIPTION
**Pull request by kjetil-kilhavn**
I just added the two simple cases for date and time variables, it will take a little more work to add all the numeric types.

### CODE REVIEW

File: src/zcl_mustache_render.clas.abap [hunk 1]
```diff
@@ -125,6 +125,8 @@ CLASS ZCL_MUSTACHE_RENDER IMPLEMENTATION.
 
     data: lr      type ref to data,
           lv_type type c.
+    data lv_date type datum.
+    data lv_time type tims.
```
Comment (mbtools) [inline src/zcl_mustache_render.clas.abap]: type d
type t 
😉

**Review by sbcgua — CHANGES_REQUESTED** [review]
@kjetil-kilhavn Thanks for the contribution. The change looks good. I agree with @mbtools comment, please change the time to internal ones. Also it would be great to add a small unit test that would cover the new feature. The simplest one possible would do.

**Comment by kjetil-kilhavn** [comment]
I was a bit unsure about the unit test. First I added the new tests to the existing find_value() method, but I later decided to add a new method instead. I hope you don't mind me using a slightly different naming convention there.

**Review by sbcgua — APPROVED** [review]
Thank you ! :+1:
Merging ...

**Pull request merged**